coverage.out

```diff
@@ -0,0 +1,25 @@
+mode: set
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:20.119,24.2 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:27.137,28.27 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:28.27,30.3 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:31.2,32.16 2 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:32.16,34.3 1 0
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:36.2,44.50 2 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:48.72,49.20 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:49.20,52.41 3 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:52.41,54.4 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:57.2,59.3 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:63.166,64.53 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:64.53,66.3 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:68.2,68.54 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:68.54,72.3 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:74.2,74.46 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:74.46,78.3 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:80.2,81.73 2 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:81.73,83.3 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:85.2,89.43 4 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:89.43,90.35 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:90.35,92.18 2 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:92.18,94.13 2 0
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:96.4,99.6 1 1
+github.com/prebid/prebid-server/v3/adapters/pixfuture/pixfuture.go:103.2,103.28 1 1
```

Comment:
There's no need to incorporate this file into our repository. Can you please delete `coverage.out`?

Reply:
fixed